examples/11_particle_dataframe.py

```diff
@@ -28,8 +35,21 @@
 df = electrons.to_df(np.s_[:100])
 print(df)
 
-# get optimal chunks
-for chunk in electrons["position"]["x"].available_chunks():
-    stride = np.s_[chunk.offset[0]:chunk.extent[0]]
-    df = electrons.to_df()
+
+# Dask example: process large datasets
+if found_dask:
+    df = electrons.to_dask()
     print(df)
+
+    # trigger a couple of compute actions
+    #   example1: average momentum in z
+    print("<momentum_z>={}".format(df["momentum_z"].mean().compute()))
+
+    #   example2: momentum histogram
+    h, bins = da.histogram(df["momentum_z"], bins=50, range=[-8.0e-23, 8.0e-23])
+    #                      weights=df["weighting"]
```

Comment:
Some issue if I pass this argument alone the lines "`Series` has no attribute `chunks`" deep inside Dask...

Reply:
It's a `dask.dataframe.core.Series` that does not have the chunks...

Reply:
Adding `.to_dask_array()` fixes this